apps/server/src/modules/files-storage/files-storage.config.ts

```diff
@@ -0,0 +1,13 @@
+import { Configuration } from '@hpi-schul-cloud/commons';
+
+import { ICoreModuleConfig } from '@src/core';
+
+export interface IFileStorageConfig extends ICoreModuleConfig {
```

Comment:
I want a way to combine one to many interfaces without re-write the key. A solution for it reduce the code a lot in future.

Reply:
I am not sure if type IFileStorageConfig  = ICoreModuleConfig & ... & ... is a way that work.

Reply:
Done by changing linter settings to allow single extends.